### ISSUE TITLE
Make some user preferences configurable

### ISSUE THREAD

**Issue by andrefpf** [open-pulse/OpenPulse#256]
The user may want to configure the font size, set custom colors for the renderer background, the points color and things like that.
We should implement this behaviour (that was already implemented for the old renderer).

My sugestion is to create a dataclass that contains all the parameters, instantiate it somewhere in the code (probably in Application or MainWindow) and make the interface access this object whenever it needs to update the renderer or the theme.

```python
from molde.colors import Color, colornames

@dataclass
class InterfacePreferences:
    background_color_0: Color = colornames.BLACK
    background_color_1: Color = colornames.WHITE
    renderer_points_color: Color = colornames.YELLOW
```

**Comment by andrefpf** [comment]
@R0drigoSchwartz was this function already implemented and merged?
If it was you can close this issue.

**Comment by R0drigoSchwartz** [comment]
Yes, it has been implemented and merged into v2.0. I will close.
